x-pack/plugin/inference/src/internalClusterTest/java/org/elasticsearch/xpack/inference/integration/AuthorizationTaskExecutorIT.java

```diff
@@ -120,9 +120,15 @@ protected Collection<Class<? extends Plugin>> getPlugins() {
         return pluginList(ReindexPlugin.class, LocalStateInferencePlugin.class);
     }
 
+    @Override
+    protected boolean resetNodeAfterTest() {
+        return true;
+    }
+
     public void testCreatesEisChatCompletionEndpoint() throws Exception {
         assertNoAuthorizedEisEndpoints();
 
+        webServer.clearRequests();
```

Comment:
A single static mock webserver is shared across all tests because the server needs to be started (listening on a port) prior to the elasticsearch node being started. So clearing the requests prior to asserting that a request was sent to ensure we aren't being affected by other tests.